src/applications/personalization/view-dependents/components/ViewDependentsSidebar/ViewDependentsSidebarBlockStates/ViewDependentSidebarBlockStates.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export const firstSidebarBlock = {
+  heading: 'When do I need to declare or modify a dependent?',
+  content: (
+    <>
```

Comment:
Might not be as important in this case, but returning these as fragments creates a flat markup structure where header and content are all in one div inside a content block, instead of being housed in separate containers. Again, probably not so important for small blocks like this though.